src/pages/nav.js

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from 'react';
+
+import {
+  MDBNavbar,
+  MDBNavbarBrand,
+  MDBNavbarNav,
+  MDBNavItem,
+  MDBNavLink,
+  MDBNavbarToggler,
+  MDBCollapse,
+  MDBDropdown,
+  MDBDropdownToggle,
+  MDBDropdownMenu,
+  MDBDropdownItem,
+  MDBContainer,
+  MDBBtn,
+} from 'mdbreact';
+
+const NavbarPage = () => {
+  const [state, setState] = useState({
+    isOpen: false,
+  });
```

Comment:
You better use
const [isOpen, setIsOpen] = useState(false);
This is the best practice with react hooks the other way was for the days of react components